README.md

```diff
@@ -137,7 +137,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [Kwanko](https://kwanko.com/) [:rocket:](https://www.kwanko.com/about-us/careers/) | An International leader in Crossdevice Performance Marketing. | `Lisboa` |
 | [Laminar](https://laminarprojects.com/) [:rocket:](https://laminarprojects.com/join-us/) | Building world class apps for the construction industry. | `Lisboa` |
 | [NILG.AI](http://www.nilg.ai/) [:rocket:](http://www.nilg.ai/) | Transforming data into actionable knowledge through AI. | `Porto` |
-
```

Comment:
Can you keep the empty line that was removed?

In order to keep consistency with other sections two empty lines should be present.

Reply:
done